docs/guides/vault-as-secret-store.md

```diff
@@ -148,12 +148,17 @@ kubectl create ns crossplane-system
 
 helm repo add crossplane-stable https://charts.crossplane.io/stable --force-update
 
-helm upgrade --install crossplane crossplane-stable/crossplane --namespace crossplane-system \
-  --set 'args={--enable-external-secret-stores}' \
-  --set-string customAnnotations."vault\.hashicorp\.com/agent-inject"=true \
-  --set-string customAnnotations."vault\.hashicorp\.com/agent-inject-token"=true \
-  --set-string customAnnotations."vault\.hashicorp\.com/role"=crossplane \
-  --set-string customAnnotations."vault\.hashicorp\.com/agent-run-as-user"=65532
+
+cat << EOF > uxp-values.yaml
```

Comment:
We'll need to remove this mention of UXP as well.

Reply:
@natereid72 This and one other reference to UXP need to be removed before this can be merged.